docs/architecture.md

```diff
@@ -284,16 +284,14 @@ Then the design for `DataContainer` interface will be like:
         ...
 
     DataConfig(
-        components={
-            "load_dataset": {
-                "type": "test_dataset",
-                "params": {"test_value": "test_value"},
-            },
-            "dataloader": {
-                "type": "_test_dataloader",  # This is the key to get dataloader
-                "params": {"test_value": "test_value"},
-            },
-        }
+        "load_dataset_config": {
```

Comment:
since this is in the python api, this needs to be `load_dataset_config = ..., ....`

Reply:
Fixed.